create-app.ts

```diff
@@ -152,6 +152,7 @@ export async function createApp({
   }
 
   if (
+    template === "streaming" &&
```

Comment:
Why is that not true for Multi-Agent?

Reply:
ah, looks like i hacked this to by pass an error, but it's great to add this for multi-agent app